### ISSUE TITLE
Impossible to import ICS from a specific website

### ISSUE THREAD

**Issue by pagiraud** [nextcloud/calendar#2029]
Hi,
I encounter a bug with Nextcloud Calendar. I often use a website, [Doctolib](https://www.doctolib.fr) to make medical appointments. Once this is done, I receive a mail with an ICS file attached to it. I can easily export this as an event to a variety of online calendars (e. g. Google Calendar), but when I try to save it in Nextcloud… nothing happens. No error message, just nothing.

You will find such a file attached to this bug report, so you can investigate (just rename .txt to .ics)
[rdv-dr-damien-abadie-27-02-2020-a-15h20.txt](https://github.com/nextcloud/calendar/files/4258639/rdv-dr-damien-abadie-27-02-2020-a-15h20.txt)
.

### Steps to reproduce
1. In Nextcloud Calendar, go to Import calendar.
2. Select the ICS file and import it to an existing calendar.

### Expected behaviour
An event should be added in the existing calendar.

### Actual behaviour
Nothing happens.

### Calendar app
**Calendar app version:** (see apps admin page, e.g. 2.0.1)
I don't have access to this information. Nextcloud version is 18.

**CalDAV-clients used:** (Thunderbird Lightning, DAVx5, Evolution, macOS Calendar, etc)
DAVx5, web interface. Both fail.

### Client configuration
**Browser:** (e.g. Firefox 48)
Firefox 73

**Operating system:** (e.g. Arch Linux)
Kubuntu, Android.

### Server configuration
All I know is there : https://scan.nextcloud.com/results/90b6bd42-f11d-438e-b582-6b16004a10cc

By the way, thank you for this great app !

**Comment by tcitworld** [comment]
The exported file doesn't respect [the specification](https://www.kanzaki.com/docs/ical/action.html). Basically it defines an alarm for the event but doesn't say what to do (sending an email, showing a notification…).

```
Validation error in iCalendar: ACTION MUST appear exactly once in a VALARM component
```

I'm guessing Google Calendar is especially tolerant to this issue. It would be nice to contact Doctolib about this issue so they can fix it.

**Comment by Ezwen** [comment]
Hi there. Just wondering, couldn't nextcloud simply ignore this VALARM in such situation?

**Comment by tcitworld** [comment]
Ideally, yes. See https://github.com/nextcloud/server/issues/25671#issuecomment-780392659

**Comment by freechelmi** [comment]
I really have the same problem and I can't even find a way to contact Doctolib on their website :-) 

I guess They all use the same bugged Ical libray from Ruby https://about.doctolib.fr/tech-doctolib/

**Comment by freechelmi** [comment]
Doctolib have been contacted , ticket number :  T-2578222

**Comment by pagiraud** [comment]
> Doctolib have been contacted , ticket number : T-2578222

They also opened a ticket for me back in 2020. They simply answered my question was out of scope (too technical)… I hope this time they will solve the issue.
